### PR TITLE
Branch update3

### PR DESCRIPTION
**Pull request by azriellee**
fixess #351 ,#354

### CODE REVIEW

File: docs/images/UserGuide/cmd enter name.png
Comment (lil1n) [inline docs/images/UserGuide/cmd enter name.png:1]: i think image name better to not contain spaces?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -105,9 +105,9 @@ University of Singapore\Desktop\TP jar file"
 ![cmd prompt open.png](images/UserGuide/cmd prompt open.png)
 5. Type `java -jar rainyDay.jar` in the terminal and press Enter on your keyboard to start the application. You should 
 see the following if the start up is successful.
-![cmd enter name.png](images/UserGuide/cmd enter name.png)
+![cmd start up.png](images/UserGuide/cmd start up.png)
```
Comment (lil1n) [inline docs/UserGuide.md:108]: i think the image will not be shown on the website as the path name contain spaces
https://talk.commonmark.org/t/whitespace-in-image-paths/1121

File: docs/UserGuide.md [hunk 2]
```diff
@@ -484,87 +587,47 @@ Suppose you want to find out what food you ate on a particular day, you can use
...
-|000007|Fried chicken                                | -$22.00      |Food and Drinks      |26/03/2023|
+|000004|GST refund                                   | +$500.00     |grants               |30/03/2023|
 +------+---------------------------------------------+--------------+---------------------+----------+
 |Viewing all entries from 2023-02-28 till today                                                      |
-|Total Inflow: $52020.00                                                                             |
-|Total Outflow: $539.00                                                                              |
-|Remaining value: $51481.00                                                                          |
+|Total Inflow: $500.00                                                                               |
+|Total Outflow: $517.00                                                                              |
+|Remaining value: $-17.00                                                                            |
```
Comment (lil1n) [inline docs/UserGuide.md:620]: i think current version is `-$` instead of `$-`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -573,15 +636,12 @@ Done, edited entry 2 from the financial report
...
-|000007|Fried chicken                                | -$22.00      |Food and Drinks      |26/03/2023|
+|000004|GST refund                                   | Ignored      |grants               |30/03/2023|
 +------+---------------------------------------------+--------------+---------------------+----------+
 |Viewing all entries from 2023-02-28 till today                                                      |
-|Total Inflow: $52020.00                                                                             |
-|Total Outflow: $539.00                                                                              |
-|Remaining value: $51481.00                                                                          |
+|Total Inflow: $0.00                                                                                 |
+|Total Outflow: $517.00                                                                              |
+|Remaining value: $-517.00                                                                           |
```
Comment (lil1n) [inline docs/UserGuide.md:644]: i think current version is `-$` instead of `$-`

File: docs/UserGuide.md (hunk 1 above)
Comment (azriellee) [reply docs/UserGuide.md:108]: oh ok i see, will amend it to have no spaces, thanks!

**Pull request merged**